packages/backend/src/controllers/v2/SemanticLayerController.ts

```diff
@@ -218,42 +218,47 @@ export class SemanticLayerController extends BaseController {
      */
     @Middlewares([allowApiKeyAuthentication, isAuthenticated])
     @SuccessResponse('200', 'Success')
-    @Get('/saved/{uuid}')
+    @Get('/saved')
     @OperationId('getSavedSemanticViewerChart')
     async getSavedSemanticViewerChart(
-        @Path() uuid: string,
-        @Path() projectUuid: string,
         @Request() req: express.Request,
+        @Path() projectUuid: string,
+        @Query() slug?: string,
+        @Query() uuid?: string,
```

Comment:
Curious about the reason to support both `slug` and `uuid` via query params instead of just `slug` via url param 🤔 is it because the tiles are working with `uuid` and therefore we need to support both?

Reply:
correct. its mix of both in all over the code. same with sql runner

Reply:
do you have a better suggestion? I was not 100% onboard with using query params. but also was lazy enough to split two routes into 4

Reply:
I think that it looks cleaner this way, also it feels like it's extra work just to do the same thing, we can probably "standardize" the way we do this between `semantic-viewer` and `sql-runner` a bit later 👌